daskhub/Chart.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: daskhub
+version: 0.0.1-set.by.chartpress
+description: Multi-user JupyterHub and Dask deployment.
+dependencies:
+  - name: jupyterhub
+    version: "0.9.1"
+    repository: 'https://jupyterhub.github.io/helm-chart/'
+    import-values:
+      - child: rbac
+        parent: rbac
+  - name: dask-gateway
+    version: "0.8.0"
+    repository: 'https://dask.org/dask-gateway-helm-repo/'
+maintainers:
+  - name: Jacob Tomlinson (Met Office)
+    email: jacob.tomlinson@informaticslab.co.uk
```

Heh this is a little out of date now.

```suggestion
  - name: Jacob Tomlinson (NVIDIA)
    email: jtomlinson@nvidia.com
```